pwnlib/gdb.py

```diff
@@ -631,6 +631,9 @@ def attach(target, gdbscript = '', exe = None, gdb_args = None, ssh = None, sysr
         if context.os == 'android':
             pre += 'set gnutarget ' + _bfdname() + '\n'
 
+        if exe:
```

Comment:
Might be useful to have a `and not context.native` or similar

Reply:
This is all inside of an `if not context.native:` clause, so I thought it would be redundant.